connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -234,6 +243,7 @@ private static Object castValueToType(Schema schema, Object value, Schema.Type t
             // Ensure the type we are trying to cast from is supported
             validCastType(inferredType, FieldType.INPUT);
 
+            log.trace("Casting to {}", targetType);
```

Comment:
I'm not sure this is worth it, if we're logging messages for each field being converted, plus if the TransformationChain has log messages with the records.